docs/_data/portland-2021-schedule.yaml

```diff
@@ -0,0 +1,73 @@
+writing_day:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:30'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - time: '9:45'
+    title: Team up and start working
+  - time: '12:30'
+    title: Lunch Break
```

Comment:
Should probably include a `Return from lunch` or something to note the timing, but not a huge deal.